labs/scorex.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: default
+title: ScoreX
+parent: Labs
+---
+# Lab Name
+ScoreX
+
+# Short Description
+
+ScoreX is a modular blockchain framework built with flexibility in mind to accelerate the design of experimental DLT technologies. ScoreX is the hyperflexible answer to the Fabric and Sawtooth frameworks. ScoreX is developed and maintained by IOHK and the founders of Ergo under the Apache 2.0 license.
```

Comment:
"ScoreX is the hyperflexible answer to the Fabric and Sawtooth frameworks." seems unnecessarily adversarial and at odd with the much better and more complete positioning that is done below with regard to all the different DLT frameworks we currently have at Hyperledger.

Reply:
It should probably say "complementary" which is the intention.

Reply:
thanks